### PR TITLE
docs: create runbook to backport changes to older versions

### PR DESCRIPTION
**Pull request by kdoroszko-splunk**
This PR contains runbook with backporting changes to older versions.

### CODE REVIEW

File: runbooks/backporting-changes-to-older-version.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Runbook to backport changes to previous versions of `addonfactory-workflow-addon-release` 
+`addonfactory-workflow-addon-release` is utilized by all supported TAs. While it is strongly recommended to use the latest minor version of the reusable action, not all TAs consistently follow this guideline. As a result, there are cases when crucial updates introduced in the latest reusable workflow version need to be backported to its older versions that are still in use.
+
+This runbook shows a real example of backporting changes correlated to `ta-automation-k8s-manifests`. In the example:
```
Comment (artemrys) [inline runbooks/backporting-changes-to-older-version.md:4]: Love the real example!

File: runbooks/backporting-changes-to-older-version.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  ```
+- changes made in `ta-automation-k8s-manifests` were correlated with changes in `addonfactory-workflow-addon-release`, so there is a need to backport **only necessary** changes to v.4.16. There are two ways to do that: by cherrypicking specific commits (and resolving the conflicts if they exist) or by commiting necessary changes manually.
+  - for example make necessary changes and commit them: 
+    ```
+    git add .github/workflows/reusable-build-test-release.yml
+    git commit -m "fix: bump k8s-manifest version"
+    ```
+- push newly created branch to the remote repository
+  - ```git push -u origin fix/bump-k8s-manifest-version```
+- in github UI:
```
Comment (artemrys) [inline runbooks/backporting-changes-to-older-version.md:34]:
```suggestion
- in GitHub UI: 
```

**Review by artemrys — COMMENTED** [review]
LGTM!

File: runbooks/backporting-changes-to-older-version.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Runbook to backport changes to previous versions of `addonfactory-workflow-addon-release` 
+`addonfactory-workflow-addon-release` is utilized by all supported TAs. While it is strongly recommended to use the latest minor version of the reusable action, not all TAs consistently follow this guideline. As a result, there are cases when crucial updates introduced in the latest reusable workflow version need to be backported to its older versions that are still in use.
+
+This runbook shows a real example of backporting changes correlated to `ta-automation-k8s-manifests`. In the example: 
+- current version of `addonfactory-workflow-addon-release` is `v.4.17.0`
+- there was a bug in `ta-automation-k8s-manifests` affecting all TAs
+  - there is a need to make a fix and release `v.4.17.1` containing fixed `ta-automation-k8s-manifests` version ([PR](https://github.com/splunk/addonfactory-workflow-addon-release/pull/329))
```
[the remark this answers is not in the document]
Comment (kdoroszko-splunk) [reply runbooks/backporting-changes-to-older-version.md:7]: `v4.17.1`, well spoted!

**Review by mkolasinski-splunk — APPROVED** [review]

**Pull request merged**